### PR TITLE
feat(pci.projects.new): manage creation errors

### PR DESCRIPTION
**Pull request by Jisay**

### CODE REVIEW

File: packages/manager/modules/pci/src/error/error.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<div class="pci-projects-new-error row justify-content-md-center mb-5">
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/error/error.html]: did `pci-projects-new-error` already exist ?

File: packages/manager/modules/pci/src/error/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "pci_error_title": "Oops ...!",
+  "pci_error_button_cancel": "Annuler",
+  "pci_error_button_action_pay_debt": "Régler ma dette",
+  "pci_error_button_action_contact_support": "Contacter le support",
+  "pci_error_default": "Une erreur est survenue lors du chargement de la page.",
+  "pci_error_account_not_eligible": "Malheureusement vous ne pouvez pas créer de projet Public Cloud car vous avez eu des projets fermés pour impayés ou pour une mauvaise utilisation du service. Nous vous invitons à contacter le support.",
+  "pci_error_max_projects_limit_reached": "Malheureusement vous ne pouvez pas créer de projet Public Cloud supplémentaire car vous avez atteint le quota maximum de projets. Pour débloquer ces quotas, nous vous invitons à contacter le support.",
+  "pci_error_paypal_account_not_verified": "Malheureusement vous ne pouvez pas créer de projet Public Cloud avec votre compte PayPal car celui ci n'est pas vérifié. Nous vous invitons à faire les démarches nécessaires auprès de PayPal pour vérifier votre compte, puis de réessayer. Vous pouvez aussi modifier <a href=\"{{ href }}\">votre moyen de paiement par défaut</a>.",
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/error/translations/Messages_fr_FR.json]:
```suggestion
  "pci_error_paypal_account_not_verified": "Malheureusement vous ne pouvez pas créer de projet Public Cloud avec votre compte PayPal car celui-ci n'est pas vérifié. Nous vous invitons à faire les démarches nécessaires auprès de PayPal pour vérifier votre compte, puis de réessayer. Vous pouvez aussi modifier <a href=\"{{ href }}\">votre moyen de paiement par défaut</a>.",
```

File: packages/manager/modules/pci/src/error/error.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        </div>
+
+        <h1 class="oui-heading_4"
+            data-translate="pci_error_title">
+        </h1>
+
+        <oui-message
+            data-type="error">
+            <span data-translate="{{ 'pci_error_' + $ctrl.error.code }}"
+                  data-translate-default="{{ 'pci_error_default' | translate }}"
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/error/error.html]:
```suggestion
                  data-translate-default="{{:: 'pci_error_default' | translate }}"
```

File: packages/manager/modules/pci/src/error/error.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+            <a class="oui-button oui-button_primary oui-button_icon-right"
+               data-ng-if="['account_not_eligible', 'unpaid_debts', 'max_projects_limit_reached'].indexOf($ctrl.error.code) > -1"
+               data-ng-href="{{ :: $ctrl.getActionButtonHref($ctrl.error.code) }}">
+                <span data-ng-bind="$ctrl.getActionButtonText($ctrl.error.code)"></span>
+                <span class="oui-icon oui-icon_circle oui-icon-chevron-right"
+                      aria-hidden="true">
+                </span>
+            </a>
+
+            <a data-ng-href="{{ $ctrl.getCancelHref() }}"
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/error/error.html]:
```suggestion
            <a data-ng-href="{{:: $ctrl.getCancelHref() }}"
```

File: packages/manager/modules/pci/src/error/error.routing.js [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+        message: {
+          type: 'any',
+        },
+      },
+      resolve: {
+        breadcrumb: () => null,
+        error: /* @ngInject */ ($transition$) => {
+          const stateParams = $transition$.params();
+          return {
+            code: snakeCase(stateParams.code),
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/error/error.routing.js]:
```suggestion
{
  ...stateParams,
  code: snakeCase(stateParams.code)
}
```

File: packages/manager/modules/pci/src/error/error.routing.js [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+            context: stateParams.context,
+            message: stateParams.message,
+          };
+        },
+        getActionButtonHref: () => (errorCode) => {
+          switch (errorCode) {
+            case 'unpaid_debts':
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/error/error.routing.js]: Possible to make constants ? :)

File: packages/manager/modules/pci/src/error/error.routing.js [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+                PCI_REDIRECT_URLS,
+                `${EnvironmentService.Environment.region}.support`,
+              );
+            default:
+              return '';
+          }
+        },
+        getActionButtonText: /* @ngInject */ $translate => (errorCode) => {
+          switch (errorCode) {
+            case 'unpaid_debts':
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/error/error.routing.js]: Constants ? :)

File: packages/manager/modules/pci/src/projects/new/new.scss [hunk 8]
```diff
@@ -0,0 +1,7 @@
+.pci-projects-new {
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/new/new.scss]: Is is useful ?

Comment (Jisay) [reply packages/manager/modules/pci/src/projects/new/new.scss]: For the moment, no :-)

**Comment by FredericEspiau** [comment]
Good job 💪

File: packages/manager/modules/pci/src/error/error.html (hunk 3 above)
Comment (Jisay) [reply packages/manager/modules/pci/src/error/error.html]: Does not work when using one time binding :-$

File: packages/manager/modules/pci/src/error/error.routing.js (hunk 6 above)
Comment (Jisay) [reply packages/manager/modules/pci/src/error/error.routing.js]: Noted. It is planned to change that logic, it will be done at that moment.

**Pull request merged**